cypress/e2e/projects.cy.js

```diff
@@ -125,22 +131,25 @@ describe('Projects', () => {
 
     // Download preview
     cy.getCy('project_nav_preview').click()
-    cy.wait(6000)
     cy.getCy('illustrated-message_format-not-supported').should('exist')
     cy.screenshot('application/projects/list/detail/preview/download')
 
     // Reset template after screenshots
     cy.getCy('project_nav_settings').click()
     cy.get('.TypeHintInput__Value a > .fa').click()
-    cy.get('.form-actions-dynamic .btn').contains('Save').click()
-
+    cy.get('.form-actions-dynamic .btn').should('be.visible').contains('Save').click()
+  })
 
-    // # Documents ------------------------------------------------------------
+  it('Documents', () => {
+    cy.loginAs('researcher')
+    cy.visitApp('/projects/c66ab9be-dd94-4dbd-92a5-ceb31658a99b')
+    cy.collapseSidebar()
 
     // Document list
     cy.getCy('project_nav_documents').click()
     cy.get('.list-group-item').should('exist')
-    cy.get('.dropdown-toggle').last().click()
+    cy.get('.dropdown-toggle').last().click({ force: true })
+    //cy.get('.dropdown-menu.dropdown-menu-right').should('be.visible')
```

Comment:
Should this be there?

Reply:
this is about the commented line right?

Reply:
I thought we were talking about line 151. Sorry about that—I’ll remove line 152

Reply:
Yes, I meant the comment.